.github/workflows/check-format.yml

```diff
@@ -28,11 +28,14 @@ jobs:
         with:
           python-version: '3.x'
 
-      # C format: clang-format-8
+      # C format: clang-format-16
       # Python format: yapf==0.22.0
       - name: Install packages
         run: |
-          sudo apt-get install -y clang-format-8
+          sudo apt-get install -y gnupg2 software-properties-common
+          wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+          sudo add-apt-repository "deb http://apt.llvm.org/focal/ llvm-toolchain-focal-16 main"
```

Comment:
Q) if we use 22.04, use `jammy` not `focal`?

Reply:
It's mistake.